cloud/cloud.go

```diff
@@ -90,6 +94,7 @@ func getVirtualMachine(username string) string {
 	if err != nil {
 		log.Fatal(err)
 	}
+	debug.Log("gateway.devbox.sh auth response: %s", string(bytes))
```

Comment:
I dunno if this would've helped when debugging, but you can also log the abbreviated stderr output inside of `client.Exec` with `debug.Log(string(exerr.Stderr))`.